java-security/src/main/java/com/sap/cloud/security/json/DefaultJsonObject.java

```diff
@@ -47,6 +47,19 @@ public boolean isEmpty() {
 		return getJSONArray(name).map(jsonArray -> castToListOfType(jsonArray, type)).orElse(Collections.emptyList());
 	}
 
+	@Override
+	public List<String> getAsStringList(String name) {
+		List<String> list = new ArrayList<>();
+		if(contains(name)) {
+			if (getJsonObject().get(name) instanceof String) {
+				list.add(getAsString(name));
+			} else {
+				list = getJSONArray(name).map(jsonArray -> castToListOfType(jsonArray, String.class)).orElse(Collections.emptyList());
```

Comment:
You could use getAsList in this line, e.g.  list = getAsList(name, String.class)

Reply:
yes, you're right... i forgot that after i've moved it from getAsList method into a the new getAsStringList() :-)